warehouse/forms.py

```diff
@@ -75,6 +84,21 @@ def __call__(self, form, field):
             raise ValidationError(msg)
 
 
+class PreventHTMLTagsValidator:
+    """
+    Validate the field to ensure that it does not contain any HTML tags.
+    """
+
+    def __init__(self, message: str | None = None):
+        if message is None:
+            message = "HTML tags are not allowed"
+        self.message = message
+
+    def __call__(self, form: BaseForm, field: Field):
+        if escape(field.data) != field.data:
```

Comment:
This is going to catch a lot of things that aren't HTML, basically any use of `<`, `>`, `'`, `"`, or `&` will effectively be prohibited from being included. I think that might be too broad?

Reply:
I've updated to use nh3 on only HTML strings.